aicsimageio/readers/default_reader.py

```diff
@@ -43,6 +43,15 @@ class DefaultReader(Reader):
     ----------
     image: types.PathLike
         Path to image file to construct Reader for.
+    dim_order: Optional[str]
+            Optional string of dimension short names for the image to use
+            instead of guess.
+            Must provide the same number of dimensions as read.
+            Default: None (guess)
+    channel_names: Optional[List[str]]
+        Optional list of channel names.
```

Comment:
this is indented differently than the above one?

Reply:
Good catch! Will fix.